css/src/components/hero.scss

```diff
@@ -34,6 +34,52 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 		}
 	}
 
+	.hero-content-right {
```

Comment:
can we come up with a name that doesn't involve a direction? assuming this element always appears in hero-content, then you're right to namespace it. maybe `hero-content-card`, not I'm married to it if you have other ideas

Reply:
Digesting with  teammates now on a better naming.  Its smelly to me too.

Reply:
`.hero-content-details`, possibly?

Reply:
@BenDMyers - thats a good option

Reply:
Currently it's a sibling of `hero-content` rather than being within `hero-content`, so it could be `.hero-details` or move it into hero-content.

Reply:
If we move it in the  hero-content, it will likely require refactoring that left content into its own node.  So I will run with `.hero-details` and get it refactored.  If this doesnt work out we can rethinking the nesting.  Thanks all.